extension-base/ctp-extension-gcf-adapter.js

```diff
@@ -1,7 +1,7 @@
 // Google Cloud Functions
 
 module.exports = {
-  ctpGcfWrapper: function(fn) {
+  ctp-extension-wrapper: function(fn) {
```

Comment:
Wait. This should be camelcased. The filenames only are dasherized.

Reply:
Oops, I realized my mistake as well but only fixed it in the azure adapter 😢